methods.go

```diff
@@ -205,12 +207,77 @@ func decodeTorrentsStatusResponse(resp *DelugeResponse) (map[string]*TorrentStat
 	return result, nil
 }
 
+// TorrentError is a tuple of a torrent and an error message, returned by
+// methods that manipulate many torrents at once.
+type TorrentError struct {
+	// ID is the hash of the torrent that experienced an error
+	ID string
+	// Message is a string message
+	Message string
+}
+
+// Error satisfies the error interface
+func (t TorrentError) Error() string {
+	return fmt.Sprintf("<%s>: '%s'")
+}
+
+// RemoveTorrents tries to remove multiple torrents at once. If successful
+// an empty list is returned with a nil error.
+// If errors were encountered, the resulting list will be a list of
+// TorrentErrors.
+//
+// Will also try to delete all downloaded files for the torrents, If
+// `rmFiles` is set, otherwise they will need to be deleted manually later.
+func (c *Client) RemoveTorrents(ids []string, rmFiles bool) ([]TorrentError, error) {
+	var torrents rencode.List
+	for i := range ids {
+		torrents.Add(ids[i])
+	}
```

Comment:
Tried to use a `torrents.Add(ids...)`, but interface{} seems to interfere with variadic functions?

Reply:
It could be related to https://dev.to/pauljlucas/go-tcha-when-nil--nil-hic